BowOpenAPI/BowOpenAPICommand.swift

```diff
@@ -0,0 +1,17 @@
+//  Copyright © 2019 The Bow Authors.
+
+import ArgumentParser
+
+struct BowOpenAPICommand: ParsableCommand {
+    static var configuration = CommandConfiguration(commandName: "bow-openapi",
+                                                    abstract: "Generate a Swift network client from an OpenAPI / Swagger specification file." )
+    
+    @Option(help: "The name for the output module.")
+    var name: String
+    
+    @Option(help: "The path to schema open api. ex. `/home/schema-openapi.json`.")
```

Comment:
We will lost the `placeholder` giving information about the parameter, for example:
`--schema <schema json|yaml>`

Now, it will be:
`--schema <schema>`

Could you use the `name` option to given this context (<schema json|yaml>, instead to <schema>) ?

Reply:
we could add this info in the help message, WDYT?

Reply:
Also, I would like to see it in the parameter use information (if it is possible, maybe using `@Option(name:...)`)

Reply:
The argument you need is: `valueName`